MANIFEST.in

```diff
@@ -0,0 +1,5 @@
+include API-Volontaria/README.rst
+recursive-include API-Volontaria/source/apiVolontaria *
+recursive-include API-Volontaria/source/location *
+recursive-include API-Volontaria/source/order *
+recursive-include API-Volontaria/source/volunteer *
```

Comment:
I think your problem is here. Why not just do that ?

```
include API-Volontaria/README.rst 
recursive-include API-Volontaria/source *
```

Reply:
I still need to add the path:

from apiVolontaria.settings import *

ModuleNotFoundError: No module named 'apiVolontaria'